### DISCUSSION TITLE
Lua variable for `'./"%e"'`

### DISCUSSION THREAD

**Discussion by ousia** [orbitalquark/textadept#296] [Q&A] [answered]
Given this macro for the current file name, without extension:

``` lua
textadept.run.run_commands.foo = './"%e"'
```

How could I get the Lua variable for this, so I  could get the variable displayed in the following command?

``` lua
ui.print('./%e')
``` 

Many thanks for your help.

**Comment by orbitalquark** [comment, marked as answer]
You would have to manually create that value. Here's how Textadept does it: https://github.com/orbitalquark/textadept/blob/393bbcdedd9c77129a3e0100a90493efb20767a5/modules/textadept/run.lua#L148-L152

**Reply by ousia** [reply]
Many thanks for your reply, @orbitalquark.
